.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  env: {
+    browser: true,
+  },
+  extends: [
+    'eslint-config-digitalbazaar/vue',
+  ],
+};
```

Comment:
Removed them, but AirBnb's lint actually requires them makes it easier to additional items in the future.

Reply:
I can't seem to think what the engineers in AirBnb were thinking about this one. I don't buy that it's "easier". In the traditional case, you add a comma (since there isn't a trailing comma) and then add the next entry.  In the trailing comma case, you add the next entry (since it's preceded by a trailing comma) and then add a trailing comma. In both cases, you need to add an entry and a comma! Now the "easier" argument comes into play if you add an entry and then don't include the trailing comma... but then that would defeat the purpose of the rule, right? In my eyes, it looks like someone was going to add an additional item and forgot.

Also, can we update this in the eslint-config-digitalbazaar package too to prevent the trailing commas proliferation? I think it's not too late.

Reply:
what happens in linting rules is one thing, but I object to changing any rule related to allowing trailing comma in regular js code.  The premise of allowing trailing comma is that it can lead to cleaner diffs, not saving someone from having to type a comma.  So, with code that already has a trailing comma in place in an array or an object, means that the diff can be for just one line if a single property or array member is added.

Reply:
you can not allow trailing commas and your grief is now an issue:
https://github.com/digitalbazaar/eslint-config-digitalbazaar/issues/17